### DISCUSSION TITLE
How to set RLS "ALL" operation in the new console?

### DISCUSSION THREAD

**Discussion by d-mok** [supabase/supabase#2019] [Questions] [answered]
After the RLS console upgrade, there is no way to set "ALL" operation for RLS.

![image](https://user-images.githubusercontent.com/49301824/122666233-bab06780-d1de-11eb-8f01-84ad783043a0.png)

Is this intentional? I used to set "ALL" operation, but now I have to set it 4 times (select, insert, update, delete).
I hope we can have the "ALL" back.

**Comment by joshenlim** [comment, marked as answer]
hey @d-mok! Ahh gotcha on this - realised we left out the ALL operation when we shipped this update! Will add in the "ALL" operation!
